coalib/parsing/LineParser.py

```diff
@@ -148,12 +157,26 @@ def __get_section_name(self, line):
 
         return ''
 
-    def __extract_keys_and_value(self, line):
+    def __extract_keys_and_value(self,
+                                 line,
+                                 delimiters,
+                                 return_value_nonempty=False):
+        """
+        This method extracts the keys and values from the give string by
+        splitting them based on the delimiters provided.
+        :param line:                  The input string.
+        :param delimiters:            A list of delimiters to split the
+                                      strings on.
+        :param return_value_nonempty: If no delimiter is found and this is true
```

Comment:
why not call this `return_second_part_nonempty` like in `__separate_by_first_occurrence` - it's only passed to `__separate_by_first_occurrence`